core/src/test/java/org/infinispan/security/CacheAuthorizationTest.java

```diff
@@ -91,6 +96,35 @@ public Void run() {
       });
    }
 
+   public void testAdminCombinations() throws Exception {
```

Comment:
Method `testAllCombinations` does test methods that require ADMIN permission, see https://github.com/infinispan/infinispan/blob/55682f391d0f6e28dd6be4ab780cb9e62a639fbd/core/src/test/java/org/infinispan/security/SecureCacheTestDriver.java#L78

The real difference between `testAdminCombinations` and `testAllCombinations` is that `testAdminCombinations` tests `EmbeddedCacheManager` actions, so it should be in a separate class `CacheManagerAuthorizationTest`.

Reply:
`CacheAuthorizationTest`, `CacheManagerAuthorizationTest` now extends `BaseAuthorizationTest`. They have the same setup

@danberindei ready for one more review